src/coreclr/jit/importer.cpp

```diff
@@ -18003,9 +18175,17 @@ void Compiler::impImportBlock(BasicBlock* block)
             }
             else if (genActualType(tree->gtType) == TYP_INT && lvaTable[tempNum].lvType == TYP_I_IMPL)
             {
-                // Spill clique has decided this should be "native int", but this block only pushes an "int".
-                // Insert a sign-extension to "native int" so we match the clique.
-                verCurrentState.esStack[level].val = gtNewCastNode(TYP_I_IMPL, tree, false, TYP_I_IMPL);
+// Spill clique has decided this should be "native int", but this block only pushes an "int".
+// Insert a sign-extension to "native int" so we match the clique.
+#ifdef TARGET_LOONGARCH64
```

Comment:
Please remove this. There should be no architecture-specific code in the importer.

And `SetContained` can't be called until Lowering.

Reply:
had removed.
Thanks